tests/phpunit/tests/functions/deleteoptionfreshsite.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Tests for _delete_option_fresh_site function.
+ *
+ * @group functions.php
+ *
+ * @covers ::_delete_option_fresh_site
+ */
+class Tests_Functions_DeleteOptionFreshSite extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57191
+	 */
+	public function test__delete_option_fresh_site() {
+		$current_option = get_option( 'fresh_site' );
+		update_option( 'fresh_site', '1' );
+
+		_delete_option_fresh_site();
+		$this->assertEquals( 0, get_option( 'fresh_site' ) );
+
+		update_option( 'fresh_site', $current_option );
```

Comment:
This should be reset before the assertion in case of test failure. Instead, use:

```php
$actual = get_option( 'fresh_site' );

update_option( 'fresh_site', $current_option );

$this->assertSame( '0', $actual );

Reply:
done